package.json

```diff
@@ -11,8 +11,7 @@
     "docs": "yuidoc",
     "docs-deploy": "yuidoc && ghp-import -r origin -p -b gh-pages docs/",
     "start": "ember server",
-    "test": "ember test",
-    "check-style": "./node_modules/jscs/bin/jscs ."
+    "test": "./node_modules/jscs/bin/jscs . && ember test"
```

Comment:
I would leave these separate, checking style is not same as testing and there may be reasons to do these tasks individually.